tests/unit/specification/grammar_test.py

```diff
@@ -41,7 +41,8 @@ def parse_declaration(data: str) -> decl.Declaration:
 def parse_formal_declaration(data: str) -> decl.Declaration:
     parser_declaration, filename = parse(data, lang.GrammarRule.session_parameter_rule)
     assert isinstance(parser_declaration, lang.FormalDecl)
-    declaration = create_formal_declaration(parser_declaration, ID("Package"), filename)
+    error = RecordFluxError()
+    declaration = create_formal_declaration(parser_declaration, ID("Package"), filename, error)
     assert isinstance(declaration, decl.Declaration)
     return declaration
```

Comment:
`error.propagate()`  should be called after `create_*`, otherwise any error in `create_*` will be just ignored.

Reply:
Fixed